tools/precommit/docstrings.py

```diff
@@ -345,7 +345,6 @@
         "get_pillars",
         "expand_variables",
         "render_jinja",
-        "expand_classes_in_order",
```

Comment:
Why is this removed?

Reply:
Because I added a docsting to `expand_classes_in_order`.

When I looked up this function name in a global grep to see if any API need to be changed I found it there and I thought it was a good thing to do but I can remove that commit if it's inappropriate.

Reply:
Oh, my bad... this is right